src/libstd/sys/common/backtrace.rs

```diff
@@ -36,7 +36,7 @@ pub fn log_enabled() -> bool {
     }
 
     let val = match env::var_os("RUST_BACKTRACE") {
-        Some(..) => 2,
+        Some(x) => if x.eq("disabled") { 1 } else { 2 },
```

Comment:
I have no idea, should I change it to `==` ? I currently don't know the difference...